src/bsoncxx/lib/CMakeLists.txt

```diff
@@ -54,8 +58,8 @@ set(bsoncxx_sources "${bsoncxx_sources}" PARENT_SCOPE)
 # Generate private headers.
 if(1)
     configure_file(
-        bsoncxx/v_noabi/bsoncxx/config/private/config.hh.in
-        bsoncxx/v_noabi/bsoncxx/config/private/config.hh
+        bsoncxx/private/config/config.hh.in
+        bsoncxx/private/config/config.hh
     )
 endif()
```

Comment:
Aside: What's up with the `if(1)` blocks all over the CMake code?

Reply:
It is a primitive workaround to make up for the lack of "scope blocks" in CMake without needing to resort to `function()` (overkill for these use cases) or comment-based begin+end delimiters. It is comparable to function-local block statements in C/C++ without any condition or loop.

That being said, this particular `if(1)` isn't doing anything useful anymore, so I've removed it in response.

Note: CMake 3.25 added [`block()`](https://cmake.org/cmake/help/v3.25/command/block.html) which would be a much better and more appropriate command to use than `if(1)` once the minimum CMake required version is raised accordingly.